modules/generator/processor/hostinfo/config.go

```diff
@@ -0,0 +1,21 @@
+package hostinfo
+
+import (
+	"flag"
+	"time"
+)
+
+type Config struct {
+	// HostIdentifiers defines the list of resource attributes used to derive
+	// a unique `grafana.host.id` value. In most cases, this should be [ "host.id" ]
+	HostIdentifiers []string      `yaml:"host_identifiers"`
+	StaleDuration   time.Duration `yaml:"stale_duration"`
+	// MetricName defines the name of the metric that will be generated
+	MetricName string `yaml:"metric_name"`
+}
+
+func (cfg *Config) RegisterFlagsAndApplyDefaults(string, *flag.FlagSet) {
+	cfg.HostIdentifiers = []string{"host.id"}
```

Comment:
Did we consider adding other resource attributes like `k8s.node.uid`, `k8S.node.name`, or `host.name`?

Reply:
We can add `k8s.node.name` to this default list as well but this is a configurable property so we can define what we want at the time of usage.